packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -16376,6 +16376,100 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+    delete:
+      tags:
+        - purposeTemplates
+      summary: Delete a Draft Purpose Template
+      description: Delete a Draft Purpose Template
```

Align with other previous calls
```suggestion
      summary: Delete a Purpose Template in Draft state
      description: Delete a Purpose Template that is in Draft state
```